README.md

```diff
@@ -1,12 +1,27 @@
-# Tsickle - TypeScript to Closure Annotator [![Build Status](https://travis-ci.org/angular/tsickle.svg?branch=master)](https://travis-ci.org/angular/tsickle)
+# Tsickle - TypeScript to Closure Translator [![Build Status](https://travis-ci.org/angular/tsickle.svg?branch=master)](https://travis-ci.org/angular/tsickle)
 
-Tsickle processes TypeScript and adds [Closure Compiler]-compatible JSDoc
-annotations. This allows using TypeScript to transpile your sources, and then
+Tsickle converts TypeScript code into a form acceptible to the [Closure Compiler].
+This allows using TypeScript to transpile your sources, and then using
 Closure Compiler to bundle and optimize them, while taking advantage of type
 information in Closure Compiler.
 
 [Closure Compiler]: https://github.com/google/closure-compiler/
 
+## What conversion means
+
+A (non-exhaustive) list of the sorts of transformations Tsickle applies:
+
+- inserts Closure-compatible JSDoc annotations on functions/classes/etc
+- converts ES6 modules into `goog.module`/`goog.provide`
```

Comment:
@evmar For the sake of people coming in from outside, I think that sentence mentioning goog.* is fantastic. It is a critical thing to understand about what tsickle does and does not do.  If I were to give somebody a super concise high-level description of tsickle, I would probably say:

Tsickle extends the TypeScript compiler to emit JavaScript code containing Closure-Compiler-compatible JSDoc-like comment annotations translated from the original types. It  can emit either ES5 or ES2016 JavaScript output, but only in Closure "goog.module" modules, not in ES2016 modules.